src/accel/LocalTransporter.cc

```diff
@@ -50,7 +51,9 @@ inline Real3 convert_from_geant(G4ThreeVector const& vec, double units)
  */
 LocalTransporter::LocalTransporter(SetupOptions const& options,
                                    SharedParams const& params)
-    : auto_flush_(options.max_num_tracks), max_steps_(options.max_steps)
+    : auto_flush_(options.max_num_tracks)
+    , max_steps_(options.max_steps)
+    , hit_manager_{params.hit_manager()}
```

Comment:
`HitManager` is shared across threads (it's called by the shared "action" inside the stepping loop), but it also needs to be individually accessed by each thread because of how Geant4 deallocates memory... so making it accessible from the local transporter was the best way I could think of doing it.

Reply:
Even though the "loop" is stored by each stepper, the actions are all shared in coreparams.